play-scala-tls-example/play

```diff
@@ -77,7 +77,7 @@ export AGENT=$(pwd)/$(find target -name 'jetty-alpn-agent-*.jar' | head -1)
 echo "Detected ALPN Agent: $AGENT "
 
 # Run Play (this should work for both run and start).
-export SBT_OPTS="$SBT_OPTS -J-javaagent:$AGENT"
+export SBT_OPTS="$SBT_OPTS -javaagent:$AGENT"
```

Comment:
This will break users of the other sbt runner...

Instead add `-J-javaagent:$AGENT` to the invocation of `sbt` (or to `JVM_OPTIONS`).

Reply:
this was an oversight I didn't intend on committing.

Reply:
```suggestion
export SBT_OPTS="$SBT_OPTS -J-javaagent:$AGENT"
```